supply/lib/supply/setup.rb

```diff
@@ -40,34 +39,58 @@ def store_metadata(listing)
       end
     end
 
-    def download_images(listing)
+    def download_images(listing, image_format = "png", remove_alpha = "false")
       UI.message("🖼️  Downloading images (#{listing.language})")
 
-      # We cannot download existing screenshots as they are compressed
-      # But we can at least download the images
       require 'net/http'
 
-      IMAGES_TYPES.each do |image_type|
-        if ['featureGraphic'].include?(image_type)
-          # we don't get all files in full resolution :(
-          UI.message("📵  Due to a limitation of the Google Play API, there is no way for `supply` to download your existing feature graphic. Please copy your feature graphic to `metadata/android/#{listing.language}/images/featureGraphic.png`")
-          next
-        end
+      allowed_imagetypes = [Supply::IMAGES_TYPES, Supply::SCREENSHOT_TYPES].flatten
 
+      allowed_imagetypes.each do |image_type|
         begin
           UI.message("Downloading `#{image_type}` for #{listing.language}...")
 
-          url = client.fetch_images(image_type: image_type, language: listing.language).last
-          next unless url
+          urls = client.fetch_images(image_type: image_type, language: listing.language)
+          next unless urls
+
+          if urls.length > 0
+            image_counter = 1
+            urls.each do |url|
+              url = "#{url}=w6000-h6000" # blowing up the sizes intentionally. So, Google would return the original image sizes as they are uploaded.
```

Comment:
6000 is a magic number here?

```suggestion
              url = "#{url}=w6000-h6000" # using huge dimentions intentionally so Google returns the images in their original size they were uploaded
```

Reply:
Just picked some random number! Too bad for someone who is actually using image sizes over this. They would get 'compressed' images then :D

Reply:
I guess Google offers no param to just get the original files?

Reply:
I think so. Because the file are in `webp` format, we can request any size. Figured if I use a big dimension, they would only return what they have.